.solcover.ts

```diff
@@ -1,10 +1,11 @@
 export default {
-  // skipFiles: [
-  //     "registry",
-  //     "v1"
-  //   ],
   skipFiles: [
-    // "v2/connectors/test/",
+    "v2/connectors/test",
```

Comment:
```suggestion
    "v2/connectors/test/*",
```

Reply:
This isn't working too. Even tried with this:

![Screenshot from 2022-06-25 23-51-46](https://user-images.githubusercontent.com/76250660/175785918-8fc195fd-a15c-48f6-ab9a-bc6cc072f7f7.png)

Checked solutions to similar [issues](https://github.com/sc-forks/solidity-coverage/issues/632#issuecomment-911780963) but none worked. Not sure if it's some issue with my system/configuration or issue with solidity-coverage.